examples/morphir-elm-projects/evaluator-tests/src/Morphir/Examples/App/SdkBasicTests.elm

```diff
@@ -1840,3 +1840,56 @@ sdkComposeRightTest ctx =
                 f not not not
         in
         g False
+
+basicsCeilingTest : Float -> Int
```

Comment:
I was under the impression that that standard didn't apply to test methods called from Scala tests - partly because I saw other newly-added ones not following it. 

(I added ~150 of them for another PR; no problem adding them here.)